qiskit/ignis/mitigation/measurement/filters.py

```diff
@@ -176,5 +182,188 @@ def fun(x):
 
             raw_data2 = new_count_dict
         else:
+            # TODO: should probably change to:
+            # raw_data2 = raw_data2[0].tolist()
             raw_data2 = raw_data2[0]
         return raw_data2
+
+
+class TensoredFilter():
+    """
+    Tensored measurement error mitigation filter
+
+    Produced from a tensored measurement calibration fitter and can be applied
+    to data
+
+    """
+
+    def __init__(self, cal_matrices, qubit_list_sizes, indices_list):
+        """
+        Initialize a tensored measurement error mitigation filter using
+        the cal_matrices from a tensored measurement calibration fitter
+
+        Args:
+            cal_matrices: the calibration matrices for applying the correction
+            qubit_list_sizes: the lengths of the lists in mit_pattern
+            (see tensored_meas_cal in circuits.py for mit_pattern)
+            indices_list (list of lists): for a calibration matrix and a state
+            label, the index of the row and column of the label
+        """
+
+        self._cal_matrices = cal_matrices
+        self._qubit_list_sizes = qubit_list_sizes
+        self._indices_list = indices_list
```

Comment:
Add a command to expose this (like the state labels for the complete meas filter).